pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -1814,6 +1814,23 @@ let
         };
       };
 
+      fill-labs.dependi = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "dependi";
+          publisher = "fill-labs";
+          version = "0.7.1";
```

```suggestion
          version = "0.7.2";
```
0.7.2 is available upstream

---

Thanks for the feedback. When I make this change, shall I amend the commit and PR title to 'unit at 7.1.2'?

---

Yes. Title will need update. And it should be a single commit.

---

Thanks, I have updated the PR. Let me know if anything needs to be modified.